klc-cert-manager/PROJECT

```diff
@@ -0,0 +1,12 @@
+domain: keptn.sh
+layout:
+- go.kubebuilder.io/v3
+projectName: klc-cert-manager
+repo: github.com/keptn/lifecycle-toolkit/klc-cert-manager
+resources:
+- controller: true
+  domain: keptn.sh
+  group: klc-cert-manager
+  kind: Deployment
+  version: v1alpha1
```

Comment:
```suggestion
  version: v1alpha2
```

Reply:
I think this should actually stay in v1alpha1. It's a different API group and those groups can progress through versions independently from each other